File: Module 4/Python/andre/hist.py
#import dependencies
import pandas as pd
import matplotlib
#import data and save to variable
df = pd.read_csv('andre.csv')
#clean data
df = df[df.Year > 1976]
df = df[df.Year < 1994]
#make visualization
df.hist("H", bins=100)
matplotlib.pyplot.show()

File: Module 4/R/andre/hist.R
#load tidyverse
library(tidyverse)
#import data
df <- read.csv("andre.csv")
#clean data
df <- filter(df, Year!= 1976)
df <- filter(df, Year < 1994)
#create histogram
ggplot(data = df, aes(df$H)) + geom_histogram()
